client/asset/btc/btc.go

```diff
@@ -536,6 +536,24 @@ type ExchangeWallet struct {
 	findRedemptionQueue map[outPoint]*findRedemptionReq
 }
 
+// ExchangeWalletSPV embeds a ExchangeWallet, but also provides the Rescan
+// method to implement asset.Rescanner.
+type ExchangeWalletSPV struct {
+	*ExchangeWallet
+}
```

Comment:
I don't know how often we'll get away with this kind of thing or the `NewAddress` dummy method, but we can try it out for a while and if it becomes hard to deal with we can switch back to explicitly set traits as described in https://github.com/decred/dcrdex/pull/1374#issuecomment-1016840825